### PR TITLE
Pool Unit Tests

### PR DESCRIPTION
**Pull request by ejwessel**
Coverage
```
--------------|----------|----------|----------|----------|----------------|
File          |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
--------------|----------|----------|----------|----------|----------------|
 contracts/   |      100 |      100 |      100 |      100 |                |
  Imports.sol |      100 |      100 |      100 |      100 |                |
  Pool.sol    |      100 |      100 |      100 |      100 |                |
--------------|----------|----------|----------|----------|----------------|
All files     |      100 |      100 |      100 |      100 |                |
--------------|----------|----------|----------|----------|----------------|
```

### CODE REVIEW

File: source/pool/test/Pool-unit.js [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+        feeToken.address,
+        {
+          from: aliceAddress,
+        }
+      )
+
+      emitted(trx, 'Withdraw', e => {
+        return (
+          e.roots[0] ===
+            '0x0000000000000000000000000000000000000000000000000000000000000000' &&
```
Comment (ejwessel) [inline source/pool/test/Pool-unit.js:120]: the variable within the `pool` `withdraw` function 
```
bytes32[] memory rootList = new bytes32[](claims.length);
```
is not used.

**Comment by ejwessel** [comment]
`81daeae44445c4d956e8d083f381169ea3b5d925` adds the ability to calculate over multiple tokens.

File: source/pool/contracts/Pool.sol [hunk 2]
```diff
@@ -134,6 +137,19 @@ contract Pool is Ownable {
...
+    external
+    view
+    returns (uint256[] memory)
+  {
+    uint256[] memory outputAmounts = new uint256[](tokens.length);
+    for (uint256 i = 0; i < tokens.length; i++) {
+      uint256 output = calculate(score, tokens[i]);
+      outputAmounts[i] = output;
+    }
+    return outputAmounts;
```
Comment (dmosites) [inline source/pool/contracts/Pool.sol:150]: If you name the return value in the signature, you don't need to explicitly return AFAIK. Would there be a benefit?

File: source/pool/contracts/Pool.sol [hunk 3]
```diff
@@ -73,6 +74,7 @@ contract Pool is Ownable {
    */
   constructor(uint256 _scale, uint256 _max) public {
     require(_max <= MAX_PERCENTAGE, "MAX_TOO_HIGH");
+    require(_scale <= MAX_SCALE, "MAX_TOO_HIGH");
```
Comment (dmosites) [inline source/pool/contracts/Pool.sol:77]: `SCALE_TOO_HIGH`

File: source/pool/contracts/Pool.sol (hunk 2 above)
Comment (ejwessel) [reply source/pool/contracts/Pool.sol:150]: Only style AFAICT. I'll change if you like.

File: source/pool/contracts/Pool.sol (hunk 3 above)
Comment (ejwessel) [reply source/pool/contracts/Pool.sol:77]: Thank you. I was struggling with `MAX_SCALE_TOO_HIGH` and hated it.

**Review by dmosites — APPROVED** [review]
LGTM

**Pull request merged**